src/eterna/AnnotationManager.ts

```diff
@@ -440,12 +433,14 @@ export default class AnnotationManager {
      * Recomputes the display objects of all annotations and layers
      */
     public updateAnnotationViews(): void {
+        // TODO: Is this necessary?
```

Comment:
This might not be necessary anymore

Reply:
Do you remember why you reset the selection highlights originally?

Reply:
I stand corrected, this I believe was necessary when annotations were selected via the AnnotationPanel, which required us to update the highlights accordingly show show/hide base ranges associated with specific annotations